kpatch-build/create-diff-object.c

```diff
@@ -2605,7 +2647,8 @@ void kpatch_create_mcount_sections(struct kpatch_elf *kelf)
 					list);
 		if (rela->type != R_X86_64_NONE ||
 		    strcmp(rela->sym->name, "__fentry__"))
-			ERROR("function '%s' has no fentry call; unable to patch", sym->name);
+			ERROR("%s: the first relocation is not for a fentry call",
+			      sym->name);
```

Comment:
Yes, I am checking the updated patch now, will update this pull request if everything is OK.